src/Credentials/CredentialProvider.php

```diff
@@ -50,9 +50,10 @@ class CredentialProvider
     /**
      * Create a default credential provider that first checks for environment
      * variables, then checks for the "default" profile in ~/.aws/credentials,
-     * then tries to make GET Request to fetch credentials if Ecs environment
-     * variable is presented, and finally checks for EC2 instance profile
-     * credentials.
+     * then checks for "profile default" profile in ~/.aws/config (which is
```

Comment:
Did you update the documentation that talks about default credential resolution to talk about this new behavior?